getItems.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.IO;
+using Azure.Data.Tables;
+using Azure;
+using System.Web;
+
+namespace TableStorage
+{
+
+
+    public class HelloAzuriteTableStorage
+    {
+
+        string AccountName = Environment.GetEnvironmentVariable("AccountName");
+        string TableItem = Environment.GetEnvironmentVariable("TableItem");
+        string Uri = Environment.GetEnvironmentVariable("Uri");
+        string AccountKey = Environment.GetEnvironmentVariable("AccountKey");
+
+        private readonly TableClient _itemTableClient;
+        public HelloAzuriteTableStorage() 
+        {
+            this._itemTableClient = 
+            new TableClient(new Uri(Uri), 
+                TableItem, 
+                new TableSharedKeyCredential(AccountName, AccountKey));
+        }
+
+        [FunctionName("PostItem")]
+        public async Task<IActionResult> RunPostItem(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            try
+            {
+
+                string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+                log.LogInformation("Request body is {requestBody}", requestBody);
+                /*
+                expect this (within postman):
+                {
+                    "PartitionKey": "3",
+                    "RowKey": "3",
+                    "Name": "HB Pencil",
+                    "Description": "It's a pencil",
+                    "Price": 2,
+                    "Quantity": 31,
+                    "isFavorite": false
+                }
+                */
```

Comment:
Nice to have an example :) Maybe move this to the read me along with some of the body shapes in the other APIs?